front/src/modules/timesStops/utils.ts

```diff
@@ -20,32 +21,25 @@ export const formatSuggestedViasToRowVias = (
       (op) => op.uic === origin.uic && op.ch === origin.ch && op.name === origin.name
     );
     // If the origin is in the ops and isn't the first operational point, we need to move it to the first position
-    if (originIndexInOps !== (-1 || 0)) {
+    if (originIndexInOps !== -1) {
```

Comment:
It seems like we should do the same test for destination.

On the screen below, I did a path between St Lazare and Dieppe by selecting both origin/via by map click, and the destination is not the last in the list as they both have the same `positionOnPath`

![Capture d’écran 2024-07-12 à 09 00 06](https://github.com/user-attachments/assets/c3aa1c6e-efdc-4c80-9ea2-843004057b37)

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/8037